app/models/commissioning_document.rb

```diff
@@ -43,7 +40,7 @@ def mime_type
   private
 
   def template
-    TEMPLATE_TYPES[@template_name].new(data_request: @data_request)
```

Comment:
Hey Damien, I'm good thanks! 
The @ at the start of a variable signifies that it is an instance variable (@@ is used for class variables).

This class is now inheriting from ActiveRecord which wires it up to a DB table and adds getters for the fields in the table, so we can use `template_name` and `data_request`.

I could have used actually `template_name` previously because I have defined a getter in line 18 (`attr_accessor :template_name`), but to get the data request I needed to use the @ version.